functions.php

```diff
@@ -153,7 +156,8 @@ function validateFields(array $post): bool {
     || (!isset($post['constitution']) || $post['constitution'] === '')
     || (!isset($post['intelligence']) || $post['intelligence'] === '') 
     || (!isset($post['wisdom']) || $post['wisdom'] === '') 
-    || (!isset($post['charisma']) || $post['charisma'] === '')) {
+    || (!isset($post['charisma']) || $post['charisma'] === '')
```

Comment:
you could use `empty()` instead of doing 2 checks here

Reply:
done and redid unit test